examples/src/java/org/apache/heron/examples/streamlet/SimplePulsarSourceTopology.java

```diff
@@ -1,131 +0,0 @@
-/**
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-
-package org.apache.heron.examples.streamlet;
```

Comment:
This file caused a break in the build as the pulsar library wasn't included in the project anymore.  We can add it back at a later time when/if we add a packaged pulsar spout to the contrib code base

Reply:
Want me to take a look at fixing this before the merge?

Reply:
yeah that'd be great.  It's probably the better than just throwing it away.

Reply:
Hey @nicknezis I'm still getting those Javadoc errors when building locally after your push

```
/Users/joshfischer/Source/apache/incubator-heron/examples/src/java/org/apache/heron/examples/streamlet/SimplePulsarSourceTopology.java:34: error: package org.apache.pulsar.client.api does not exist
import org.apache.pulsar.client.api.Consumer;
                                   ^
/Users/joshfischer/Source/apache/incubator-heron/examples/src/java/org/apache/heron/examples/streamlet/SimplePulsarSourceTopology.java:35: error: package org.apache.pulsar.client.api does not exist
import org.apache.pulsar.client.api.PulsarClient;
                                   ^
/Users/joshfischer/Source/apache/incubator-heron/examples/src/java/org/apache/heron/examples/streamlet/SimplePulsarSourceTopology.java:36: error: package org.apache.pulsar.client.api does not exist
import org.apache.pulsar.client.api.PulsarClientException;
                                   ^
/Users/joshfischer/Source/apache/incubator-heron/examples/src/java/org/apache/heron/examples/streamlet/SimplePulsarSourceTopology.java:52: error: cannot find symbol
    private PulsarClient client;
            ^
  symbol:   class PulsarClient
  location: class PulsarSource
/Users/joshfischer/Source/apache/incubator-heron/examples/src/java/org/apache/heron/examples/streamlet/SimplePulsarSourceTopology.java:53: error: cannot find symbol
    private Consumer consumer;
            ^
  symbol:   class Consumer
  location: class PulsarSource
5 errors
```

Reply:
To reproduce execute the below from the root directory of the project:
`cd website2/website && make javadocs`

Reply:
Weird, I originally got the error, but then after the edit it went away. I do get a long list of warnings about invalid use of tags. I'll try a fresh `git clone` and see if it fails again.

Reply:
Interesting. I was able to recreate the issue. Apparently at some point I had run `bazel build --config=darwin examples/src/java/...` and that made the problem go away. After running `bazel clean --expunge`, the issue came back. I'll keep looking into it.